applications/processor/aggregator-processor/src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+management.defaults.metrics.export.enabled=false
+spring.cloud.config.enabled=false
```

Comment:
1. To add a shared property we should do it [here](https://github.com/spring-cloud/stream-applications/blob/df9fedbd669171557cf72f23b370d514dd5fa55c/applications/stream-applications-core/pom.xml#L144) rather than proliferate a new application.yml in all of the apps.

2. We do not want to add this property as it disables the config server for all apps. Explanation follows:


The reason you are seeing the failures is because

> Spring Boot 2.4 introduced a new way to import configuration data via the spring.config.import property. This is now the default way to bind to Config Server.
[Details](https://docs.spring.io/spring-cloud-config/reference/client.html#config-data-import).

However, the stream-apps currently use the [Config First Bootstrap](https://docs.spring.io/spring-cloud-config/reference/client.html#config-first-bootstrap) approach.

To enable this we have to do one of the following:
- set `spring.cloud.bootstrap.enabled=true`
- add dep to the [spring-cloud-starter-bootstrap](https://github.com/spring-cloud/spring-cloud-commons/blob/b1f7e9c565c914a034f56c7c2c72d86545e5ada9/spring-cloud-starter-bootstrap/src/main/java/org/springframework/cloud/bootstrap/marker/Marker.java#L23)
- set `spring.config.use-legacy-processing=true`

The first is the least intrusive approach.

What you have done here in this PR is disable the config server altogether for all apps - we do not want to do that.

The "Config First Bootstrap" approach is considered legacy and we should probably have got off of it in 5.0.0 but it is too late in the 5.0.x line to change this behavior. For now, lets simply add the `spring.cloud.bootstrap.enabled=true` to the shared application properties in 

https://github.com/spring-cloud/stream-applications/blob/df9fedbd669171557cf72f23b370d514dd5fa55c/applications/stream-applications-core/pom.xml#L144

Reply:
Setting spring.cloud.bootstrap.enabled=true in the common properties seems to have done the trick.